### PR TITLE
Fixes #207

### LINKED ISSUE #207: 404 Page and 500 Page Not Edited, No Header, No Red Drop Down Menu and No Link for Fossasia Summit 2018 in Drop Down Menu
Issue 1- 404 Page and 500 Page is as same as in the Template. 
Issue 2- No Fossasia Logo and Favicon in 404 and 500 Pages
Issue 3- No Header and No Drop Down Menu in Both the Pages
Issue 4- No Links for the Fossasia summit 2018 in the Drop Down Menu

### PR DESCRIPTION
**Pull request by NaviRocker**
- Fixed all the Issues in #207

### CODE REVIEW

File: 404.html [hunk 1]
```diff
@@ -3,18 +3,44 @@
...
+        <meta name="twitter:image" content="http://fossasia.github.io/fossasia15/img/fossasia-dark.png">
+
+        <link rel="shortcut icon" href="fossasia.ico" type="image/x-icon" />
         <link href="css/bootstrap.min.css" rel="stylesheet" type="text/css" media="all"/>
         <link href="css/flexslider.min.css" rel="stylesheet" type="text/css" media="all"/>
         <link href="css/elegant-icons.min.css" rel="stylesheet" type="text/css" media="all"/>
+        <link href="css/font-awesome.min.css" rel="stylesheet" type="text/css" media="all"/>
         <link href="css/pe-icon-7-stroke.min.css" rel="stylesheet" type="text/css" media="all"/>
         <link href="css/lightbox.min.css" rel="stylesheet" type="text/css" media="all"/>
-        <link href="css/theme.css" rel="stylesheet" type="text/css" media="all"/>
```
Comment (0x48piraj) [inline 404.html:13]: What about `<link href="css/theme.css" rel="stylesheet" type="text/css" media="all"/>` 
You deleted that too :open_mouth:

**Review by 0x48piraj — COMMENTED** [review]
Work needed!

**Comment by NaviRocker** [comment]
theme.css is not required because it not used even for the index.html

**Comment by 0x48piraj** [comment]
Oh! Let me check.

> Comment in review from next time :wink:

**Comment by NaviRocker** [comment]
Is everything Fine?

**Comment by 0x48piraj** [comment]
I deployed and checked, seems kinda fine :wink:

**Comment by abishekvashok** [comment]
Link to test: https://naveentherocker.github.io/fossasia.org/

**Comment by NaviRocker** [comment]
Yes @abishekvashok

**Comment by 0x48piraj** [comment]
We'll know that @abishekvashok :relaxed: 
Anyways, :+1:

**Review by 0x48piraj — APPROVED** [review]

**Review by isuruAb — APPROVED** [review]
👍

**Comment by isuruAb** [comment]
@naveentherocker  thank you

**Pull request merged**